### DISCUSSION TITLE
Submodule Error

### DISCUSSION THREAD

**Discussion by panditdhamdhere** [Cyfrin/foundry-full-course-cu#1836] [General]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85685981/2ff3ed34-09b2-42bf-b18f-ebca58483372)
Getting this error every time I install openzeppelin and chainlink in my foundry project

**Comment by urosognjenovic** [comment]
You need 's' at the end of the name of the repo. It is ```chainlink-brownie-contracts```, not ```chainlink-brownie-contract```. Try with that and let us know if it solved the problem.

**Reply by panditdhamdhere** [reply]
hi @thebraveman17 now diffrent error 
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85685981/940b641e-4a93-45c4-bf8c-5d116546a32c)

**Reply by EngrPips** [reply]
@panditdhamdhere, This is a network issue. Please retry when you have a stable network.

**Reply by panditdhamdhere** [reply]
@EngrPips problem solved, thank you very much sir, you always here to solve our problems and support us. Truly grateful.

**Reply by EngrPips** [reply]
I am glad I could help. Keep crushing it. Friend.
